### DISCUSSION TITLE
Question About -65535

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#119] [BaiduKDDCup2022]
The criterion of abnormal values is too strict. If the prediction has relative large error in one day, then the result would be -65535(Our method performs normal offline but gets -65535 online). Maybe one of the test dataset is out of distribution. Therefore, some models return 'abnormal prediction'. Worse still, we don't know the specific cause of this error. I think it's better to only keep the necessary criterion, such as empty prediction, and nan prediction.

**Comment by zhanshijinwat** [comment]
I have the same problem. I get normal score on my computer,but get -65535 online. Maybe the rmse is too big, and acc is set to -1,  and return -65535?
